certbot/util.py

```diff
@@ -367,71 +342,13 @@ def get_var_from_file(varname, filepath="/etc/os-release"):
             return _normalize_string(line.strip()[len(var_string):])
     return ""
 
-
 def _normalize_string(orig):
     """
     Helper function for get_var_from_file() to remove quotes
     and whitespaces
     """
     return orig.replace('"', '').replace("'", "").strip()
 
-
-def get_python_os_info():
```

Comment:
I think we still need to use this function where it was previously being called. [distro does not support Windows or macOS](https://github.com/nir0s/distro/issues/177) and its [BSD support was added in version 1.3](https://github.com/nir0s/distro/blob/master/CHANGELOG.md#130-20180509) which is newer than is available in distros like Ubuntu 18.04.

The way I'd recommend doing this is adding a boolean parameter like `full_name` to this function and then in the `linux` section of this function pass it to `distro.linux_distribution`. `distro.linux_distribution(full_distribution_name=False)[:2]` is the same as the current call in `get_os_info()` and I think `distro.linux_distribution(full_distribution_name=True)` works nicely for `get_os_info_ua()`.

Reply:
I pushed the changes in. I'm currently installing FreeBSD to test this out (to confirm that the return values are indeed empty in this case amongst other things), so I would like to wait with approving until tested against that as well.

Reply:
Good catch about the BSD support btw, I completely missed it myself.

Reply:
Ok, this needs yet another refactor. Distro raises exception when imported under unsupported platforms.
```
ImportError: Unsupported platform: freebsd12
```

Reply:
This should apply to windows as well. Do you know why aren't we seeing the `ImportError` on our Azure tests?

Reply:
Yeah. Most of our tests including those on Windows are running with [distro 1.4.0](https://github.com/certbot/certbot/blob/0f31d9b7ac1bb01a3ff6ac396fe315cb657eef6f/letsencrypt-auto-source/pieces/dependency-requirements.txt#L71). The only tests that are running with an older version are [our "oldest" tests](https://github.com/certbot/certbot/blob/0f31d9b7ac1bb01a3ff6ac396fe315cb657eef6f/tools/oldest_constraints.txt#L56) which currently only run on Linux.